src/_samples/index.md

```diff
@@ -252,26 +268,37 @@ abstract class Describable {
 }
 {% endprettify %}
 
-Any class extending `Describable` has the `describeWithEmphasis()` method, which calls the extender's implementation of `describe()`.
+Any class extending `Describable` or using it as a mixin has the `describeWithEmphasis()` method, which calls the extender's / base class implementation of `describe()`.
+
+<?code-excerpt "lib/samples/spacecraft.dart (abstract use)" replace="/abstract/[!$&!]/g"?>
+{% prettify dart %}
+class Probe extends Spacecraft with Describable {
+  Probe(String name) : super(name, new DateTime.now());
+}
+{% endprettify %}
```

Comment:
I've added example example of the use of the abstract class, though as a mixin.

Reply:
LGTM. @filiph what do you think?

Reply:
I'm not sure if this alone helps explain how this works. If we're adding sample code, I'd suggest adding:

```
var probe = new Probe("probe");  // TODO(chalin): some better name
probe.describeWithEmphasis();  // Method taken from the Describable class.
```

Also not sure whether introducing a mixin in the example is a good idea. It might be confusing that we're suddenly showing a mixin.

Reply:
Ok, then I'll just pull that out.